src/components/Cardlist.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { useFetchCardsData } from '../hooks/CardContainer';
+import '../style/cardlist.css';
+import searchIcon from '../images/Search.svg';
+import noCardImg from '../images/noCardImg.png';
+
+const Cardlist = () => {
+  const data = useFetchCardsData();
```

Comment:
```suggestion
  const cardList = useFetchCardsData();
```

data 보다는 좀더 명시적으로 어떤 데이터임을 변수명을 통해 유추할 수 있게하면 가독성 + 유지보수 측면에서 관리하기 좋습니다.